packages/components/header/template.njk

```diff
@@ -78,7 +78,7 @@
     </div>{# close nhsuk-header__container #}
     <div class="nhsuk-navigation-container">
       <nav class="nhsuk-navigation" id="header-navigation" role="navigation" aria-label="Primary navigation">
-        <ul class="nhsuk-header__navigation-list">
+        <ul class="{{ "nhsuk-header__navigation-list nhsuk-header__navigation-list-fewer-links" if params.primaryLinks.length < 4 else "nhsuk-header__navigation-list" }}">
```

Comment:
Rather than adding this class automatically based on number of links, maybe it’d make sense to have it as a class that can be optionally added? (Or make `—-justify` the optional one?)

I can see there being some scenarios where you might want left-alignment even with 4 or 5 links? 🤔

Reply:
Might be worth a discussion on best approach?

If doing it automatically I'd be tempted to apply a class like `nhsuk-header_navigation-list--justified` when there are over x links. So the class sets the appearance, it doesn't describe the links.

Reply:
Hi Frankie, I’ve spoken to service manual interaction designers who agree with this method of having a hard coded limit of navigation links as it's an issue that's been raised a few times in the design system issues page - could you add your suggestion as a separate issue in this repo? It would be good to feed it back to the designers